pages/common/chafa.md

```diff
@@ -0,0 +1,25 @@
+# chafa
+
+> Image printing in the terminal.
+> See also: `catimg` and `pixterm`.
+> More information: <https://hpjansson.org/chafa>.
+
+- Print an image or animated GIF to the terminal:
+
+`chafa {{path/to/input}}`
```

Comment:
We prefer `path/to/file`
```suggestion
`chafa {{path/to/file}}`
```

Reply:
Got it. I'll update that here and in my `pixterm` and `catimg` PRs.

Reply:
Updated.